java/androidpayload/library/src/com/metasploit/meterpreter/AndroidMeterpreter.java

```diff
@@ -33,27 +33,7 @@
 import com.metasploit.meterpreter.android.webcam_start_android;
 import com.metasploit.meterpreter.android.webcam_stop_android;
 import com.metasploit.meterpreter.android.android_wlan_geolocate;
-import com.metasploit.meterpreter.stdapi.Loader;
```

Comment:
I'm not certain we can do wildcard import on stdapi and android at the same time since it causes a conflict with getuid. Let me try an experiment...